hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -52,19 +52,19 @@
 	static final int RELEASE_0140_VERSION = 3;
 	static final int MERKLE_VERSION = RELEASE_0140_VERSION;
 	static final long RUNTIME_CONSTRUCTABLE_ID = 0x8d4aa0f0a968a9f3L;
-	static final RichInstant[] NO_CONGESTION_STARTS = new RichInstant[0];
+	static final Instant[] NO_CONGESTION_STARTS = new Instant[0];
 	static final DeterministicThrottle.UsageSnapshot[] NO_SNAPSHOTS = new DeterministicThrottle.UsageSnapshot[0];
 
-	public static final RichInstant UNKNOWN_CONSENSUS_TIME = null;
+	public static final Instant UNKNOWN_CONSENSUS_TIME = null;
 
 	static DomainSerdes serdes = new DomainSerdes();
 	static Supplier<ExchangeRates> ratesSupplier = ExchangeRates::new;
 	static Supplier<SequenceNumber> seqNoSupplier = SequenceNumber::new;
 
 	private int stateVersion = UNRECORDED_STATE_VERSION;
-	private RichInstant[] congestionLevelStarts = NO_CONGESTION_STARTS;
+	private Instant[] congestionLevelStarts = NO_CONGESTION_STARTS;
 	private ExchangeRates midnightRates;
-	private RichInstant consensusTimeOfLastHandledTxn;
+	private Instant consensusTimeOfLastHandledTxn;
```

Comment:
Change to `private Instant consensusTimeOfLastHandledTxn = UNKNOWN_CONSENSUS_TIME;`

Reply:
Done.